packages/amplify-graphql-model-transformer/src/resolvers/rds/resolver.ts

```diff
@@ -216,7 +230,7 @@ export const createLayerVersionCustomResource = (scope: Construct, resourceNames
       },
       // Make the physical ID change each time we do a deployment, so we always check for the latest version. This means we will never have
       // a strictly no-op deployment, but the SQL Lambda configuration won't change unless the actual layer value changes
```

Comment:
nit: move this comment above where you're creating they physical ID so we're explaining the motivation in context.